R/pkg/R/mllib.R

```diff
@@ -71,3 +71,27 @@ setMethod("predict", signature(object = "PipelineModel"),
           function(object, newData) {
             return(dataFrame(callJMethod(object@model, "transform", newData@sdf)))
           })
+
+#' Get statistics on a model.
```

Comment:
`statistics` -> `summary`. We also need to have a better description of the return value. For example, this is R's doc of `summary.glm` (https://stat.ethz.ch/R-manual/R-devel/library/stats/html/summary.glm.html):

~~~
summary.glm returns an object of class "summary.glm", a list with components
...
coefficients	the matrix of coefficients, standard errors, z-values and p-values. Aliased coefficients are omitted.
...
~~~